waku/v2/utils/time.go

```diff
@@ -2,14 +2,13 @@ package utils
 
 import "time"
 
-// GetUnixEpoch converts a time into a unix timestamp with the integer part
-// representing seconds and the decimal part representing subseconds
-func GetUnixEpochFrom(now time.Time) float64 {
-	return float64(now.UnixNano()) / float64(time.Second)
```

Comment:
I believe this change was what was causing the time collisions in the tests on fast machines. The nanos are always 0000000 in this implementation.


<img width="528" alt="CleanShot 2022-03-09 at 17 47 30@2x" src="https://user-images.githubusercontent.com/65710/157571518-cd8ff591-c399-4130-8ac0-4d6d43a37c60.png">